app/mailers/user_mailer.rb

```diff
@@ -9,4 +9,13 @@ def rent_created(rent_id)
          subject: default_i18n_subject,
          date: Time.zone.now)
   end
+
+  def rent_expired(rent_id)
+    @rent = Rent.find(rent_id)
```

Comment:
Why define this variable as global if you are using it only localy?
Or you can do something little more DRY with memorization and combine this call with the one of the above mailer...